lib/openers/sumatra-opener.js

```diff
@@ -9,11 +9,13 @@ export default class SumatraOpener extends Opener {
     const args = [
       '-forward-search',
       `"${texPath}"`,
-      lineNumber,
+      `"${lineNumber}"`,
       `"${filePath}"`
     ]
 
-    child_process.execFile(sumatraPath, args, (error) => {
+    const command = `${sumatraPath} ${args}`
```

Comment:
The fix is pretty mandatory, I just haven't tested (as this is pretty simple edit) and it will fail to open otherwise.

I don't mind mergin and applying your fix.